src/full/Agda/Compiler/Backend/Boot.hs

```diff
@@ -0,0 +1,78 @@
+{-# LANGUAGE GADTs #-}
+{-# OPTIONS_GHC -Wunused-imports #-}
+
+module Agda.Compiler.Backend.Boot where
```

Comment:
Could this be called `Agda.Compiler.Backend.Base` please as it is not a small boot module?